mod_ci/controllers.py

```diff
@@ -464,6 +465,22 @@ def queue_test(db, gh_commit, commit, test_type, branch="master", pr_nr=0):
     from run import log
 
     fork = Fork.query.filter(Fork.github.like("%/CCExtractor/ccextractor.git")).first()
+    githubapi = requests.get("https://api.github.com/repos/ccextractor/ccextractor/pulls/{pr}".format(pr=Test(pr_nr)))
```

Comment:
Why are you still using `Test(pr_nr)`? We pass the `pr_nr` in as an argument to this function...